staging/src/k8s.io/api/apps/v1/types.go

```diff
@@ -111,6 +111,13 @@ type RollingUpdateStatefulSetStrategy struct {
 	// Default value is 0.
 	// +optional
 	Partition *int32 `json:"partition,omitempty" protobuf:"varint,1,opt,name=partition"`
+	// The maximum number of pods that can be unavailable during the update.
+	// Value can be an absolute number (ex: 5) or a percentage of desired pods (ex: 10%).
+	// Absolute number is calculated from percentage by rounding up. This can not be 0.
+	// Defaults to 1. This field is alpha-level and is only honored by servers that enable the
+	// MaxUnavailableStatefulSet feature.
+	// +optional
+	MaxUnavailable *intstr.IntOrString `json:"maxUnavailable,omitempty" protobuf:"varint,2,opt,name=maxUnavailable"`
```

Comment:
I was surprised this didn't change fixture test data under k8s.io/api/testdata. Turns out, our test generator wasn't populating *intstr.IntOrString objects. Fix is in https://github.com/kubernetes/kubernetes/pull/85258. Once that merges, rebase and run `UPDATE_COMPATIBILITY_FIXTURE_DATA=true go test ./vendor/k8s.io/api/` to update fixture data with this new field.